### DISCUSSION TITLE
How can we change the title bar color?

### DISCUSSION THREAD

**Discussion by Akascape** [martinet101/pymica#12] [Q&A] [answered]
Hello @marticliment, I really like your work on the mica theme for python oriented UI.
This might be an off-topic question but I need a little bit help with some code.

I actually want to change the **header/titlebar color** of a window (like the title bar of a simple tkinter window).

I think it can be done with `DwmSetWindowAttribute` using `DWMWA_CAPTION_COLOR`, but don't know what exactly will be the code for this.

I found a relevant example here (not written in python): https://www.purebasic.fr/english/viewtopic.php?t=78732
 
I hope there is a solution :)
**Thanks...**

**Comment by marticliment** [comment, marked as answer]
What you could do is clone the win32mica repo and then start messing with the vaules so they match the ones in the example you provided.

Let's say you edit __init__.py so it shows like this:
DWM_UNDOCUMENTED_MICA_ENTRY = 34
DWM_DOCUMENTED_MICA_ENTRY = 34
DWM_UNDOCUMENTED_MICA_VALUE = 35
DWM_DOCUMENTED_MICA_VALUE = 35


What would happen?

**Reply by Akascape** [reply]
@marticliment Thanks, it's working with `DWM_UNDOCUMENTED_MICA_ENTRY = 35 `. 
But can we pass a color name (string) instead of integers as the argument?
Maybe replacing this part with something: `ctypes.byref(ctypes.c_int(DWM_UNDOCUMENTED_MICA_VALUE))`.

**Reply by marticliment** [reply]
Looking at the docs, you should be able to pass hex values, as shown here: https://learn.microsoft.com/en-us/windows/win32/gdi/colorref.

However, i'm new to this color api, and I have no idea how does it work
